utils/firestore.js

```diff
@@ -2281,6 +2281,10 @@ const storeKitReceipt = async (package) => {
         });
 
         await participantDoc.ref.update({
+            '684635302': '375535639',
+            '254109640': '375535639',
```

Comment:
Just double checking here: the DD has yes/no values listed for these two vars. I'm not sure if we have to update the DD or if this code needs to be:
```javascript
'684635302': '353358909',
'254109640': '353358909',
```